### ISSUE TITLE
Scan across several VLAN:s

### ISSUE THREAD

**Issue by agirdea** [pucherot/Pi.Alert#199]
Hi,
Trying to scan different network segment, such as '192.168.0.0/24 10.10.0.0/24'. The 10.10.0.0/24 is on a different VLAN=10, while 192.168.0.0/24 is on the default (VLAN=1). Running the scan on individual segements works, but when used both segments and add the --vlan=10 parameter it resturns only the 10.10.x.x devices, reporting the others on the default VLAN as offline. 

Can pi-alert made to run several scans sequencially, each of it with individualt vlan tags?

**Comment by 4Arthur2** [comment]
Hi agirdea, 
I've tryed the solution from jokob-sk for untagged networks. It was working fine for me. Now I get my vlan interfaces eth0, eth0.3 .......
https://github.com/jokob-sk/Pi.Alert/commit/929ea98fcef7d8b8af81181d97a8f9dff16665fe

For getting the tagged, I've tryed with with the  -Q <VLAN ID> without success.
with arp-scan it's working eg. (arp-scan 192.168.3.0/24 --interface eth0 -Q 3) , but not in the pialert.conf 

Cu
Daniel
